### PR TITLE
add game screen

### PR DESCRIPTION
**Pull request by TarekIdrees**

### CODE REVIEW

File: app/src/main/res/drawable/rounded_check_box.xml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
```
Comment (AmeerAmjed) [inline app/src/main/res/drawable/rounded_check_box.xml:1]: why attributes have **hardcode** please change to resource value

File: app/src/main/res/drawable/rounded_game_card.xml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+
+    <solid android:color="@color/card" />
+    <corners android:radius="16dp" />
```
Comment (AmeerAmjed) [inline app/src/main/res/drawable/rounded_game_card.xml]: why attributes have hardcode please change to resource value

File: app/src/main/res/drawable/rounded_search_btn.xml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:tools="http://schemas.android.com/tools" xmlns:android="http://schemas.android.com/apk/res/android" tools:ignore="StateListReachable">
+
+    <item android:state_pressed="true">
+        <shape android:shape="rectangle">
+            <solid android:color="@color/border" />
+            <corners android:radius="16dp" />
```
Comment (AmeerAmjed) [inline app/src/main/res/drawable/rounded_search_btn.xml]: why attributes have hardcode please change to resource value

File: app/src/main/res/drawable/rounded_search_btn.xml [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+        </shape>
+    </item>
+
+    <item android:state_pressed="true">
+        <shape android:shape="rectangle">
+            <solid android:color="@android:color/transparent" />
+            <corners android:radius="16dp" />
```
Comment (AmeerAmjed) [inline app/src/main/res/drawable/rounded_search_btn.xml]: why attributes have hardcode please change to resource value

File: app/src/main/res/drawable/shape_shadow_of_game_card.xml [hunk 5]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (AmeerAmjed) [inline app/src/main/res/drawable/shape_shadow_of_game_card.xml]: Where is shadow in **Game Screen ( snafer )**?

File: app/src/main/res/layout/activity_game.xml [hunk 6]
```diff
@@ -0,0 +1,232 @@
...
+    </androidx.appcompat.widget.LinearLayoutCompat>
+
+    <androidx.appcompat.widget.AppCompatButton
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:background="@drawable/rounded_btn"
+        android:text="@string/next_word"
+        android:textAllCaps="false"
+        android:textColor="@color/on_primary"
+        android:textSize="@dimen/font_size_normal"
```
Comment (AmeerAmjed) [inline app/src/main/res/layout/activity_game.xml]: Use style font from resources

File: app/src/main/res/layout/activity_game.xml [hunk 7]
```diff
@@ -0,0 +1,232 @@
...
+
+
+        </androidx.appcompat.widget.LinearLayoutCompat>
+
+
+    </androidx.appcompat.widget.LinearLayoutCompat>
+
+    <androidx.appcompat.widget.AppCompatButton
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
```
Comment (AmeerAmjed) [inline app/src/main/res/layout/activity_game.xml]: set height `large_button_height`

File: app/src/main/res/values/colors.xml [hunk 8]
```diff
@@ -3,6 +3,7 @@
     <color name="background">@color/white</color>
 
     <color name="primary">#72BFC7</color>
+    <color name="primary_dark">#BE72BFC7</color>
```
Comment (AmeerAmjed) [inline app/src/main/res/values/colors.xml]: why value dark theme in light?????

File: app/src/main/res/values-night/colors.xml [hunk 9]
```diff
@@ -2,6 +2,8 @@
 <resources>
     <color name="background">#1F1D2B</color>
 
+    <color name="primary_dark">#BE6C5ECF</color>
```
Comment (AmeerAmjed) [inline app/src/main/res/values-night/colors.xml]: why add new value to primary_dark   ????

**Review by AmeerAmjed — CHANGES_REQUESTED** [review]

File: app/src/main/res/drawable/rounded_check_box.xml (hunk 1 above)
Comment (TarekIdrees) [reply app/src/main/res/drawable/rounded_check_box.xml:1]: Done

File: app/src/main/res/drawable/rounded_search_btn.xml (hunk 3 above)
Comment (TarekIdrees) [reply app/src/main/res/drawable/rounded_search_btn.xml]: Done

File: app/src/main/res/drawable/rounded_search_btn.xml (hunk 4 above)
Comment (TarekIdrees) [reply app/src/main/res/drawable/rounded_search_btn.xml]: Done

File: app/src/main/res/drawable/shape_shadow_of_game_card.xml (hunk 5 above)
Comment (TarekIdrees) [reply app/src/main/res/drawable/shape_shadow_of_game_card.xml]: the file has been added by the wrong and I forgot to delete it. 
Done

File: app/src/main/res/values/colors.xml (hunk 8 above)
Comment (TarekIdrees) [reply app/src/main/res/values/colors.xml]: The name should be primary_pressed. I resolve this issue 
Done

File: app/src/main/res/values-night/colors.xml (hunk 9 above)
Comment (TarekIdrees) [reply app/src/main/res/values-night/colors.xml]: by wrong it should be primary_pressed

File: app/src/main/res/layout/activity_game.xml (hunk 7 above)
Comment (TarekIdrees) [reply app/src/main/res/layout/activity_game.xml]: Done

File: app/src/main/res/layout/activity_game.xml (hunk 6 above)
Comment (TarekIdrees) [reply app/src/main/res/layout/activity_game.xml]: Done

File: app/src/main/res/drawable/rounded_game_card.xml (hunk 2 above)
Comment (TarekIdrees) [reply app/src/main/res/drawable/rounded_game_card.xml]: Done

**Review by AmeerAmjed — APPROVED** [review]
Amazing

**Review by mosayed01 — APPROVED** [review]

**Pull request merged**